src/saving/SaveManagerGUI.tscn

```diff
@@ -102,6 +97,7 @@ margin_right = 1000.0
 margin_bottom = 561.0
 size_flags_vertical = 3
 SelectableItems = true
+ErrorSaveDeletionFailedPath = null
```

Comment:
This seems incorrect to me, this should be causing errors when opening the scene.

Reply:
Hmmm. It doesn't cause any crashes, I actually can't see where the heck that property even is in the editor.

<img width="522" alt="image" src="https://github.com/Revolutionary-Games/Thrive/assets/18731181/2a157b9a-9617-40e5-8b86-cba371144020">

Not sure what's going on there... Something to do with renaming the path variable and Godot freaking out?

Reply:
Removed the line, testing - everything works as expected.

<img width="1543" alt="image" src="https://github.com/Revolutionary-Games/Thrive/assets/18731181/5b0dd0c4-1969-409e-90e1-adda5efdea59">
<img width="1029" alt="image" src="https://github.com/Revolutionary-Games/Thrive/assets/18731181/d82c10e0-28a6-4f04-a993-653225a2f5e8">

Reply:
I was commenting that in the base scene this is set, but when instanced in this other scene the value is overridden as null, which to me should cause the code for the scene to print an error because it can't find the node from the path as the NodePath is null. 

In the instanced scene node there should be a revert button next to the property which resets it back to the default value that is in the original scene. This is the way you see if a scene instance overrides the default parameters as set in its own scene in Godot.

Reply:
If you only looked at the root node of the scene, then you see the properties of that root node (and that's why you didn't see it). To see the overridden property you need to click on the instanced node name in the scene tree to look at it and not the root node of the scene.

Reply:
The screenshot posted (Showing the missing property) was inside the SaveManagerGUI scene, and I'd selected the "SaveList" that was part of the scene hierarchy there. :)

Reply:
Hmm, that should be right. Probably a more niche issue happened (like compilation failed so hot reload didn't work or something like that, or godot editor state got out of sync with the files which can very rarely happen. In that case restarting godot helps it to then properly write out the actual values that should be in the on disk files). I won't spend the time checking out an earlier version of this PR to see what's going on as this was resolved already.